app/src/main/res/values/strings.xml

```diff
@@ -671,4 +671,27 @@
     <string name="text_connected">Connected</string>
     <string name="text_disconnected">Disconnected</string>
 
+    <string name="lux_meter_description">Measures the ambient light \nintesity</string>7
```

Comment:
As @CloudyPadmal  mentioned, try to keep these string resources in the middle of the file